services/src/test/scala/cromwell/services/metadata/MetadataServiceSpec.scala

```diff
@@ -74,4 +77,94 @@ class MetadataServiceSpec extends FlatSpec with Matchers with TableDrivenPropert
     }
   }
 
+  // For the metadata event tests!
+  val eventKeyRegex = "path\\:to\\:failures\\[([0-9]*)\\]\\:message".r
```

Comment:
Yeah... I mean it's just a case of where an exception is thrown in a test, but... yeah.